app/code/AKisilenko/ModuleLesson6/Cron/Status.php

```diff
@@ -22,38 +30,67 @@ class Status
     /**
      * Status constructor.
      * @param LoggerInterface $logger
+     * @param ScopeConfigInterface $scopeConfig
      * @param CollectionFactory $askQuestionsFactory
      */
     public function __construct(
         LoggerInterface $logger,
+        ScopeConfigInterface $scopeConfig,
+//        Config $config,
```

Comment:
Remove commented code if it's not needed

Reply:
done